consensus/reactor.go

```diff
@@ -301,8 +373,14 @@ func makeRoundStepMessages(rs *RoundState) (nrsMsg *NewRoundStepMessage, csMsg *
 	return
 }
 
+func (conR *ConsensusReactor) GetRoundState() *RoundState {
+	conR.mtx.Lock()
```

Comment:
This shouldn't be needed because conR.conS.GetRoundState() locks on its own mutex?